golink.go

```diff
@@ -541,24 +544,42 @@ func expandLink(long string, env expandEnv) (*url.URL, error) {
 
 func devMode() bool { return *dev != "" }
 
+const peerCapName = "tailscale.com/golink"
```

Comment:
Currently the ACL grammar requires an `https://` prefix on the peer cap name. We intend to remove it, but for now it's still needed. (What I did in tailsql was to check for both, but that's probably overkill)

Reply:
this does work.  I'm using it currently in my personal tailnet.  I was chatting with Maisem earlier about the syntax, and we'll probably relax it a little further, but for now I think this should be okay.

Reply:
Weird. I just tested it on my tailnet and it rejected the name without it. Maybe I did something else wrong in my test. But as long as it works LGTM. 🙂